apps/xero/utils.py

```
@@ -65,25 +74,112 @@ def __init__(self, credentials_object: XeroCredentials, workspace_id: int):
+        """
+        is_import_to_fyle_enabled = False
+
+        configuration = WorkspaceGeneralSettings.objects.filter(workspace_id=self.workspace_id).first()
+        if not configuration:
+            return is_import_to_fyle_enabled
+
+        if attribute_type in ['ACCOUNT'] and configuration.import_categories:
+            is_import_to_fyle_enabled = True
+
+        elif attribute_type == 'SUPPLIER' and configuration.import_suppliers_as_merchants:
+            is_import_to_fyle_enabled = True
```

Ruff (0.11.9)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)